pkg/network/netlink/consumer.go

```diff
@@ -116,16 +116,6 @@ func NewConsumer(procRoot string, targetRateLimit int, listenAllNamespaces bool)
 	}
 	c.initWorker(procRoot)
 
-	var err error
-	c.do(true, func() {
```

Comment:
Is there a reason this was moved? Previously, we would get an error if there was problem creating the netlink socket. Now, there is only log messages and the user of the `Consumer` has no idea why. They just see the `Events()` channel closed.

Reply:
The reason I moved this was because `c.socket` is reused a bunch of times during the initial dump and so is closed (by the last dump) by the time we get into the `receive` loop. So we have to initialize `c.socket` again before we get into the loop.

I was going for a quick fix for the point release, but looks like that is a remote possibility at this point. Let me see if I can get back the old error behavior as much as possible.